chocolatey-diff/Private/Invoke-DiffTool.ps1

```diff
@@ -23,18 +23,15 @@ function Invoke-DiffTool {
 #>
     [CmdletBinding()]
     param (
-        [Parameter(Mandatory)]
-        [string] $Path1,
-
-        [Parameter(Mandatory)]
-        [string] $Path2
+        [Parameter(Mandatory)][string] $Path1,
+        [Parameter(Mandatory)][string] $Path2
     )
     $diffTool = if ($env:difftool) {
         $env:difftool
     } elseif (Test-IsUnix) {
         "diff"
     } else {
-        "C:\Program Files\KDiff3\bin\diff.exe"
+        "C:\tools\cygwin\bin\diff.exe"
```

Comment:
IMO, I think it is more likely that a windows user have `git` installed instead of `cygwin`.
So maybe defaulting to `C:\Program Files\Git\usr\bin\diff.exe` would be a better option?

Reply:
The version shipped with Git is newer than the cygwin version, so I kinda agree that it would be better.